task1/index-task1.js

```diff
@@ -0,0 +1,34 @@
+const subjects: {
+    mathematics: {
+        students: 200,
+        teachers: 6,
+    },
+    biology: {
+        students: 120,
+        teachers: 6,
+    },
+    geography: {
+        students: 60,
+        teachers: 2,
+    },
+    chemistry: {
+        students: 100,
+        teachers: 3,
+    },
+};
+console.log(subjects);
+
+const subjectsKeys = Object.keys(subjects);
+const stringSubjects = subjectsKeys.join();
+console.log(stringSubjects);
+
+const subjectsValues = Object.values(subjects);
+
+const sumValues = subjectsValues.reduce((acc, item) => acc + item.students + item.teachers, 0);
+console.log(sumValues);
+
+const averageStudents = subjectsValues.reduce((acc, item) => acc + item.students, 0) / subjectsKeys.length;
+console.log(averageStudents);
+
+const teachers = subjectsValues.sort((a, b) => b.teachers - a.teachers);
```

Comment:
It should be named something like subjectsSortedByTeachers. When i see name teacher i expect to see array of teachers or something like that